jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -38,10 +38,10 @@ pipeline {
             defaultValue: "bwc-test-opensearch-dashboards",
             trim: true
         )
-        booleanParam(
+        choice(
             name: 'BUILD_DOCKER',
-            description: 'Build docker image or not.',
-            defaultValue: true
+            description: 'Build docker image or not with options.',
+            choices: ['build_docker', 'build_docker_with_build_number_tag', 'do_not_build_docker'],
```

Comment:
Wanted to confirm if programmatically triggering this workflow might be affected by this change

Reply:
I have already taken care of all the instances that get touched by this change.
Thanks.

Reply:
Jenkins default the 1st entry in the choices list is the default, thus the behavior is exactly the same as before and I have changed all the instance to reflect this change.

Reply:
I guess you didn't get my question. I was concerned if choices and parameters have different affect on the workflow. If build-workflow triggers docker-workflow, can we select an option like default choice or something?

Reply:
Ok got it. This was new to me Thanks!